eva_submission/eload_validation.py

```diff
@@ -452,6 +522,17 @@ def _aggregation_report(self):
             reports.append('Not performed')
         return '\n'.join(reports)
 
+    def _structural_variant_check_report(self):
+        sv_dict = self.eload_cfg.query('validation', 'structural_variant_check', 'files')
```

Comment:
`structural_variant_check` is never set anywhere in this code

Reply:
I have addressed this now

Reply:
Well, `structural_variant_check` is set now but `'validation', 'structural_variant_check', 'files'` isn't

Reply:
I have set the config file now